### PR TITLE
Milestone 1: list structure

### PR DESCRIPTION
**Pull request by yonas44**
- Set up linters and webpack.
- Use webpack as a bundler for the project.
- Use JavaScript to render tasks to dynamically.

Thank you for reviewing my PR!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,52 @@
-# Todo-list_webpack
 A todo-list project built with Webpack and ES6 modules
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Todo-list_webpack
+
```
Comment (christianbiring1) [inline README.md]: - [x] Kindly make sure your project has a descriptive name `eg: To-Do List` 
- [x] Also consider adding a detailed description telling what the project is about for the repo to be documented professionally. You can refer [here](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=Estimated%20time%3A%203.5h-,Description,-In%20this%20project)

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @yonas44 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
# STATUS: REQUEST CHANGES :recycle: 
soclose

## Highlights
- [ ] All linters checks are green :heavy_check_mark: 
- [ ] Descriptive PR `title` and `summary`
- [ ] Good commit messages :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (yonas44) [reply README.md]: Okay thanks for the feedback, I will have them fixed.

File: README.md [hunk 2]
```diff
@@ -12,7 +12,11 @@ A todo-list project built with Webpack and ES6 modules
...
 ## Getting Started
 
-For this project we implemented adding and removing of books using JavaScript
+In this project I built a simple HTML list of To Do tasks using webpack and served by a webpack dev server along with ES6 modules.
+
+- The webpage will allow users to add lists easily using the input field.
+- The completed tasks will each have a line through, implying the task is complete.
+- The completed tasks can be removed all together using a button.
 
 ### Setup
```
Comment (AmaduKamara) [inline README.md]: [OPTIONAL CHANGE] 
This change won't stop your project from being approved but will help you document your project well.

You could add these steps below the clone command
- Run `git clone https://github.com/yonas44/Todo-list_webpack.git`
- Run `cd Todo-list_webpack.git`
- Run `npm install`
-  Run `npm start`

These will help anyone who wants to get your project running on their computer have access easily.

**Review by AmaduKamara — APPROVED** [review]
# Status: APPROVED 🟢🟢🟢 

Hi @yonas44,

Good job on making the changes asked by the previous reviewer.
Your project is complete! 

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ Good commit messages 👍

There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: README.md (hunk 2 above)
Comment (yonas44) [reply README.md]: Okay thanks for the feedback, I will add them. Thanks!🙏

**Pull request merged**